csharp-basics/exercises/Collections/Histogram/Program.cs

```diff
@@ -9,15 +9,32 @@ namespace Histogram
 {
     class Program
     {
-        private const string Path = "../Histogram/midtermscores.txt";
```

Comment:
Šādāveidā path vajag

Reply:
Es saprotu bet ja es nelieku pilno path tad man nestrādā kods, met error.

Reply:
Kādu tieši erroru?

Reply:
Unhandled Exception: System.IO.DirectoryNotFoundException: Could not find a part of the path 'C:\Users\aleks\source\repos\CodeLex\csharp-basics\exercises\Collections\Histogram\bin\Histogram\midtermscores.txt'.
   at System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   at System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   at System.IO.StreamReader..ctor(String path, Encoding encoding, Boolean detectEncodingFromByteOrderMarks, Int32 bufferSize, Boolean checkHost)
   at System.IO.File.InternalReadAllText(String path, Encoding encoding, Boolean checkHost)
   at System.IO.File.ReadAllText(String path)
   at Histogram.Program.Main(String[] args) in C:\Users\aleks\source\repos\CodeLex\csharp-basics\exercises\Collections\Histogram\Program.cs:line 16

Reply:
Jo tāda path nav C:\Users\aleks\source\repos\CodeLex\csharp-basics\exercises\Collections\Histogram\bin\Histogram\midtermscores.txt'. Jāsaliek parezi ../, lai meklētu īstajā mapē. Pieņemu, ka vēl vienu failu ārpus, jo tagad tu meklē iekš bin

Reply:
es nezinu kā taisīt pareizus saīsinātos Path